sesman/chansrv/clipboard.c

```diff
@@ -372,6 +378,16 @@ clipboard_init(void)
         return 0;
     }
 
+	if (0 != config_read(&g_cfg)) {
+		g_printf("clipboard: error reading config. quitting.\n");
```

Comment:
Some formatting issue : new line needed before the brace, and don't use tabs, but spaces. There is a file for coding standard at the base of the project